### ISSUE TITLE
Rendering error when scroll HtmlViewer on Linux

### ISSUE THREAD

**Issue by aberenguel** [sepinf-inc/IPED#984]
IPED versions: 3.18.12 ~and 4.0.0-snapshot~
SO: Ubuntu 21.10

Since I started to use IPED I noticed that html became buggy when I scrolled the html preview frame. See below (red blocks were added to hide the messages)

![image](https://user-images.githubusercontent.com/2973359/155763874-6eb63163-3bf8-4951-a3b6-9db8cfe804c7.png)

~Today I tried to fix it. I added one line in the HtmlViewer.java to debug,and surprisely it fixed. I think the `evento.toString()` triggered something that refreshed the rendering (just supposition).~

```java
        Platform.runLater(new Runnable() {
            @Override
            public void run() {
                htmlViewer = new WebView();
                htmlViewer.setOnScroll((event) -> System.out.println("scroll " + event));  // <<<<< the line I added
                webEngine = htmlViewer.getEngine();
                addHighlighter();

                StackPane root = new StackPane();
                root.getChildren().add(htmlViewer);

                Scene scene = new Scene(root);
                jfxPanel.setScene(scene);
            }
        });
```

~PS: Just println a string without `event` doesn't fix.~

**Comment by lfcnassif** [comment]
Hi @aberenguel. What do you mean by buggy? Is it freezing? Rendering wrong elements? Another thing? What java version are you using to open the case?

**Comment by aberenguel** [comment]
I mean the rendering of message ballons.
![image](https://user-images.githubusercontent.com/2973359/155799658-b7d38414-6291-4371-a3bf-6938e74ab9df.png)

After a lot of debug, I noticed that if I open a chat before loading the `Help.htm`, the rendering issue does not happen.
I also commented those lines to check and the rendering became ok:
```
FileProcessor exibirAjuda = new FileProcessor(-1, false);
exibirAjuda.execute();
```

**Comment by aberenguel** [comment]
> Hi @aberenguel. What do you mean by buggy? Is it freezing? Rendering wrong elements? Another thing? What java version are you using to open the case?

Java version: 1.8.0_311, vendor: Oracle Corporation

**Comment by lfcnassif** [comment]
This seems really weird, but maybe there is a better fix, I'll take a look next week, thanks for reporting this!

**Comment by lfcnassif** [comment]
> I noticed that html became buggy

Do you know if it used to work in any previous iped or java version on Linux?

**Comment by lfcnassif** [comment]
@aberenguel could you try using the Liberica OpenJDK 8 FULL (not standard) package?

**Comment by aberenguel** [comment]
> @aberenguel could you try using the Liberica OpenJDK 8 FULL (not standard) package?
> 
> PS: or Liberica OpenJDK 11 FULL with 4.0.0

Just tried some scenarios:
* IPED 3.8.12 + Liberica JDK 8 Full  → same behaviour as Oracle JDK 8
* IPED 3.8.12 + Liberica JDK 11 Full → it worked properly!!
* IPED 4.0.0 + Oracle JDK 11 + OpenJFX 11.0.2 (added in pom.xml) → the rendering ballons are ok, but the chat wallpaper is not shown.
* IPED 4.0.0 + Liberica JDK 11 Full → it worked properly!!

In the header message I mistakenly mentioned IPED 4.0.0. Sorry :)

**Comment by aberenguel** [comment]
A think I noticed when using IPED 3.8.12 + JDK 8:

* when the Help.htm is show in the first, the chat ballons rendering is buggy and the scrollbar is ok
![image](https://user-images.githubusercontent.com/2973359/156638107-c3336f82-8ed3-4916-aff7-3b1dd216ee71.png)

* when the Help.htm is NOT shown in the first, the chat ballons rendering is ok but the scrollbar is not shown
![image](https://user-images.githubusercontent.com/2973359/156638799-79bfdcf2-b447-41b4-8d56-36fcd2ea5823.png)

**Comment by lfcnassif** [comment]
Thanks @aberenguel for all your tests! Looks like a JDK 8 bug to me. Since this is fixed in 4.0.0 (planned to be released soon), I think we can close this like a few other issues fixed just in 4.0.0 (e.g. #420)

PS: If you find a reasonable fix to 3.18.x without upgrading the JDK (this could result in other unexpected issues...), I'll be happy to apply.

**Comment by lfcnassif** [comment]
Closing, already fixed in master. Thanks @aberenguel for your tests!

If anyone finds an easy solution to apply in 3.x version, please let me know.
